pom.xml

```diff
@@ -126,13 +126,14 @@
                     <version>${org.codehaus.mojo.sql-maven-plugin.version}</version>
                 </plugin>
                 <plugin>
-                    <groupId>net.revelc.code</groupId>
+                    <groupId>net.revelc.code.formatter</groupId>
                     <artifactId>formatter-maven-plugin</artifactId>
                     <version>${formatter-maven-plugin.version}</version>
                     <configuration>
                         <configFile>${project.root.basedir}/eclipse/formatter.xml
                         </configFile>
                         <lineEnding>CRLF</lineEnding>
+                        <sourceDirectory>${project.basedir}</sourceDirectory>
```

Comment:
I confirmed that the directory to format Java is only under `${project.basedir}/src/main/java` .
So I fixed with b14a2417c6cc48d9792e02f5ed760770e3bd8cfb commit .